### DISCUSSION TITLE
🚀 Table filters- make the add column button searchable

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4103] [Feature Requests]
### Suggest an idea for ExpressionEngine

I just noticed, on a site with a bunch of fields, that add column dropdown gets really hard to find things in, especially because it's not sorted by alpha.  I'd love for it to be searchable, like most dropdowns do automatically beyond a certain length.

And I'm not really requesting alpha sort, because I do think the main ones folks use should be at the top, but then maybe a section for custom fields that's alpha.  Or not- either way, search would be really handy.

And while poking table filters- I still like the idea of making any field included in the column a filter automatically.
https://github.com/ExpressionEngine/ExpressionEngine/discussions/2971

**Comment by intoeetive** [comment]
@Yulyaswan fyi

**Comment by Yulyaswan** [comment]
@robinsowell please have a look #4211
